bugsnag/django/__init__.py

```diff
@@ -45,14 +46,23 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    req = {
```

Comment:
Is `request_tab` a better name for this?

Reply:
Fixed in 4b59d6cf5ee86e11ede693f10421c1f95379d044